templates/home.html

```diff
@@ -1,81 +1,52 @@
+      
     <div>
-      <div class="marginBottom1">A community which drives our career path into DevOps.</div>
-      <div>What does than mean?</div>
-      <div class="marginBottom1">The community mission is transfering DevOps knowledge by practice. We are implementing an open source web app (www.devopsloft.io), while picking up DevOps technologies, like: git (GitHub), AWS, Linux, scripting, Jenkins, Chef/Ansible.</div>
-      <div class="marginBottom1">The goal of the community is addressing the lack of exprianced DevOps specialists to fulfil the increasing demand.</div>
-      <div>How?</div>
-      <div class="marginBottom1">Assisting you building a career path in the DevOps domain by driving DevOps knowledge transfer in a study group which develops a real webapp (www.devopsloft.io). And, matching you with potentials organizations who seek DevOps engineers with your skillset (technical and personal).</div>
-      <div class="marginBottom1">We will study together DevOps, once a week, two hours session frontally using Skype or suchlike (shared lesson). The rest of the work is done on your own. You will work on issues assigned to you. One can order personal coaching/training.</div>
-      <div class="marginBottom1">You are welcomed to attend the program as long as you feel it is valuable for you.</div>
-      <div class="marginBottom1">According to your pace, we will set you interviews with potentials organizations.</div>
-      <div class="marginBottom1">Recommended skills: A degree in Computer Science/Software Development/Industrial Engineering/related technical discipline or equivalent experience (DEV/IT/OPS) Ability to understand and write code.</div>
-      <div class="marginBottom1">What to bring:</div>
-      <div>yourself</div>
-      <div>your laptop</div>
-      <div class="marginBottom1">We will be working with services like: GitHub, AWS and others. Some of those services require payments. Those costs are on your own expense.</div>
-      <div>Costs</div>
-      <div class="marginBottom1">Participation is free of charge and commitment</div>
-      <div>Weekly Shared Lesson</div>
-      <div class="marginBottom1">Every Tuesday 10AM Israel Time via Skype</div>
-      <div>If you'd like to add a feature or correct a defect, please open a pull request.</div>
-      <div class="marginBottom1">If you have questions or suggestions, please open an issue.</div>
-      <div class="marginBottom1">And be sure to follow us on Facebook and LinkedIn.</div>
-      <div>And ... join our Facebook Group and our Meetup Group.</div>
+        <h3>What does than mean?</h3>
+        The community mission is transfering DevOps knowledge by practice. We are implementing an open source web app (www.devopsloft.io), while picking up DevOps technologies, like: git (GitHub), AWS, Linux, scripting, Jenkins, Chef/Ansible.
+        <br>
+        The goal of the community is addressing the lack of exprianced DevOps specialists to fulfil the increasing demand.
```

Comment:
same

Reply:
What does that mean?
The community mission is transferring DevOps knowledge by practice. We are implementing an open source web app (www.devopsloft.io), while picking up DevOps technologies, like: git (GitHub), AWS, Linux, scripting, Jenkins, Chef/Ansible. 
The goal of the community is addressing the lack of experienced DevOps specialists to fulfill the increasing demand.

@bregman-arie Did you mean that?

Reply:
@jkubkub the work 'than' should be 'that'. I think.

Reply:
@lioramilbaum Yep , spotted that. Missed some letters also in different words. Pushed changes already.